plugins/modules/idrac_storage_volume.py

```diff
@@ -891,7 +897,8 @@ def main():
         "raid_reset_config": {"choices": ['true', 'false'], "default": 'false'},
         "raid_init_operation": {"choices": ['None', 'Fast']},
         "job_wait": {"type": "bool", "default": True},
-        "job_wait_timeout": {"type": "int", "default": 900}
+        "job_wait_timeout": {"type": "int", "default": 900},
+        "time_to_wait": {"type": "int", "default": 300}
```

Comment:
Just run once with job_wait_timout less than 300 seconds, and see if it impacts existing code or not?

Reply:
Import scp sends to request to host to restart - this wait is the time the OS takes to honor this request. In our case, wait happens immediately after calling the import API, So this time to_wait sort of parallel to the job_wait time and not added to it.